moveit_experimental/package.xml

```diff
@@ -16,7 +16,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend version_gte="1.11.2">pluginlib</build_depend>
+  <build_depend>pluginlib</build_depend>
```

Comment:
Why did you removed/changed dependency settings? Probably, that's a merge/rebase failure. Please revert these changes.